src/test/java/org/jenkinsci/plugins/authorizeproject/strategy/SpecificUsersAuthorizationStrategyTest.java

```diff
@@ -187,20 +188,18 @@ public void testAuthenticateWithPassword() throws Exception {
         assertTrue(SpecificUsersAuthorizationStrategy.authenticate("test1", false, null, "test1"));
         assertFalse(SpecificUsersAuthorizationStrategy.authenticate("test1", false, null, "test2"));
         assertFalse(SpecificUsersAuthorizationStrategy.authenticate("test1", false, null, ""));
-        assertFalse(SpecificUsersAuthorizationStrategy.authenticate("", false, null, "test2"));

```

Comment:
(Just a note for me)
> The synthetic test case with userId="" has to be removed because the
underlying library does not support this anymore.
```
[ERROR] testAuthenticateWithApitoken(org.jenkinsci.plugins.authorizeproject.strategy.SpecificUsersAuthorizationStrategyTest)  Time elapsed: 3.188 s  <<< ERROR!
com.google.common.util.concurrent.UncheckedExecutionException: java.lang.IllegalArgumentException: Cannot pass null or empty values to constructor
        at com.google.common.cache.LocalCache$Segment.get(LocalCache.java:2234)
        at com.google.common.cache.LocalCache.get(LocalCache.java:3965)
        at com.google.common.cache.LocalCache$LocalManualCache.get(LocalCache.java:4764)
        at jenkins.security.UserDetailsCache.loadUserByUsername(UserDetailsCache.java:127)
        at hudson.model.User$UserIDCanonicalIdResolver.resolveCanonicalId(User.java:1243)
        at hudson.model.User$CanonicalIdResolver.resolve(User.java:1184)
        at hudson.model.User.get(User.java:509)
        at org.jenkinsci.plugins.authorizeproject.strategy.SpecificUsersAuthorizationStrategy.authenticate(SpecificUsersAuthorizationStrategy.java:125)
        at org.jenkinsci.plugins.authorizeproject.strategy.SpecificUsersAuthorizationStrategyTest.testAuthenticateWithApitoken(SpecificUsersAuthorizationStrategyTest.java:204)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
        at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
        at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
        at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
        at org.jvnet.hudson.test.JenkinsRule$1.evaluate(JenkinsRule.java:600)
        at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
        at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.IllegalArgumentException: Cannot pass null or empty values to constructor
        at org.acegisecurity.userdetails.User.<init>(User.java:127)
        at org.jvnet.hudson.test.JenkinsRule$DummySecurityRealm.loadUserByUsername(JenkinsRule.java:933)
        at jenkins.security.UserDetailsCache$Retriever.call(UserDetailsCache.java:172)
        at jenkins.security.UserDetailsCache$Retriever.call(UserDetailsCache.java:161)
        at com.google.common.cache.LocalCache$LocalManualCache$1.load(LocalCache.java:4767)
        at com.google.common.cache.LocalCache$LoadingValueReference.loadFuture(LocalCache.java:3568)
        at com.google.common.cache.LocalCache$Segment.loadSync(LocalCache.java:2350)
        at com.google.common.cache.LocalCache$Segment.lockedGetOrLoad(LocalCache.java:2313)
        at com.google.common.cache.LocalCache$Segment.get(LocalCache.java:2228)
        ... 21 more
```